src/main/java/ca/intelliware/ihtsdo/mlds/config/WebConfigurer.java

```diff
@@ -52,69 +49,56 @@ public class WebConfigurer implements ServletContextInitializer {
     //   java -Dspring.profiles.active=production -Dserver.keystore.file=file:///`pwd`/src/main/resources/keystore.p12 -jar target/oauth-1.0.0.BUILD-SNAPSHOT.jar
     @Bean
     @Profile("ssl")
-    EmbeddedServletContainerCustomizer containerCustomizer(
+    public WebServerFactoryCustomizer<TomcatServletWebServerFactory> containerCustomizer(
             @Value("${keystore.file}") Resource keystoreFile,
             @Value("${keystore.pass}") final String keystorePass) throws Exception {
         final String absoluteKeystoreFile = keystoreFile.getFile().getAbsolutePath();
-        
+
         log.info("Web application configuration, SSL keystore: {}", absoluteKeystoreFile);
 
-        return new EmbeddedServletContainerCustomizer() {
-			
-			@Override
-			public void customize(ConfigurableEmbeddedServletContainer container) {
-            TomcatEmbeddedServletContainerFactory tomcat = (TomcatEmbeddedServletContainerFactory) container;
-            tomcat.addConnectorCustomizers(
-                new TomcatConnectorCustomizer() {
-					@Override
-					public void customize(Connector connector) {
-	                    connector.setPort(8443);
-	                    connector.setSecure(true);
-	                    connector.setScheme("https");
-	
-	                    Http11NioProtocol proto = (Http11NioProtocol) connector.getProtocolHandler();
-	                    proto.setSSLEnabled(true);
-	                    proto.setKeystoreFile(absoluteKeystoreFile);
-	                    proto.setKeystorePass(keystorePass);
-	                    //proto.setKeystoreType("jks");
-	                    //proto.setKeyAlias("tomcat");
-					}
-                });
-			}
+        return (TomcatServletWebServerFactory factory) -> {
+            factory.addConnectorCustomizers((TomcatConnectorCustomizer) connector -> {
+                connector.setPort(8443);
+                connector.setSecure(true);
+                connector.setScheme("https");
+
+                Http11NioProtocol proto = (Http11NioProtocol) connector.getProtocolHandler();
+                proto.setSSLEnabled(true);
+//             proto.setKeystoreFile(absoluteKeystoreFile);
+//             proto.setKeystorePass(keystorePass);
+                //proto.setKeystoreType("jks");
+                //proto.setKeyAlias("tomcat");
+            });
         };
     }
-    
+
     @Override
     public void onStartup(ServletContext servletContext) throws ServletException {
         log.info("Web application configuration, using profiles: {}", Arrays.toString(env.getActiveProfiles()));
         EnumSet<DispatcherType> disps = EnumSet.of(DispatcherType.REQUEST, DispatcherType.FORWARD, DispatcherType.ASYNC);
 
         initMetrics(servletContext, disps);
-        if (env.acceptsProfiles(Constants.SPRING_PROFILE_PRODUCTION)) {
+        if (env.acceptsProfiles(Profiles.of(Constants.SPRING_PROFILE_DEVELOPMENT))) {
```

Comment:
@jaykamal when this code goes to prod, will we lose any static pages due to this condition?

Reply:
@dmcgihtsdo We have changed from development to production now.